lib/file-source/src/checkpointer.rs

```diff
@@ -110,6 +107,12 @@ impl CheckpointsView {
         }
     }
 
+    pub fn contains_bytes_checksums(&self) -> bool {
+        self.checkpoints
+            .iter()
+            .any(|entry| matches!(entry.key(), FileFingerprint::BytesChecksum(_)))
+    }
+
```

Comment:
This function looks like it just moved without change. Am I missing seeing what changed?

Reply:
Ah yes, I was going to move that because it is only used by tests, but ended up leaving it in-place. I'll put it back just to avoid the confusing diff.